src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,47 @@
+package duke.task;
+
+import duke.data.DateTime;
+
+import java.text.ParseException;
+
+/**
+ * The Event class represents any tasks that have a description, and an event date in nature.
+ */
+public class Event extends Task {
+    private String _date;
+    private DateTime _dateTime;
+
+    /**
+     * Creates an Event object, which is also a Task.
+     * @param desc a description of the Event Task
+     * @param date the date the Event will be held on
+     */
+    public Event(String desc, String date) {
+        super(desc);
+        try {
+            this._dateTime = new DateTime(date);
+            this._date = this._dateTime.getDateTimeString();
+        } catch (ParseException e) {
+            System.err.println("Cant parse Date: " + date);
```

Comment:
It'll be better if you returned a String with the feedback message instead of println, especially if you want to pass it to your GUI.
It's even better if you declare a CommandResult class.